src/Core/Components/DateTime/FluentCalendarBase.cs

```diff
@@ -40,6 +40,18 @@ public abstract class FluentCalendarBase : FluentInputBase<DateTime?>
     [Parameter]
     public bool CheckIfSelectedValueHasChanged { get; set; } = true;
 
+    /// <summary>
+    /// Defines the appearance of the <see cref="FluentCalendar"/> component.
+    /// </summary>
+    [Parameter]
+    public virtual CalendarViews View { get; set; } = CalendarViews.Days;
+
+    /// <summary>
+    /// Defines the appearance of the <see cref="FluentCalendar"/> component.
+    /// </summary>
+    [Parameter]
+    public virtual string? DateTimeFormat { get; set; } = null;
```

Comment:
I don't think it's a good idea to allow date display customization. It's better to continue to use the default OS format: `Culture.DateTimeFormat`.

This would cause too many problems when encoding or managing Accessibility.
For example, in your case, the encoding of `2006-01-01 - 2006-01-01`, which is correct, doesn't work.

![peek](https://github.com/microsoft/fluentui-blazor/assets/8350694/621dc451-caf6-4c16-a9e2-e14e685bbbd6)

Reply:
OK, I'll fugire out a better way

Reply:
Why do you want a so specific format for a Year selector? 
Why not to display "the year" like `2006` (based on the Culture)?

Reply:
Replaced to use System.Globalizarion class except for year view where I didn't find any usable and used hard coded yyyy